src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  eventElement.preventDefault();
+
+  const largePrevious = document.querySelector('.gallery__large-img');
+  largePrevious.src = eventElement.target.src;
+  
+};
```

Comment:
This function changes the source of the large image to the source of the clicked thumbnail. However, it does not include a condition to check if the clicked element is a thumbnail. This means that if a user clicks on the large image or the gallery container (but not on a thumbnail), the source of the large image will be changed to undefined, which is not the intended behavior. To fix this, you should add a condition to check if the clicked element is a thumbnail before changing the source of the large image.